### PR TITLE
removed physical createdir, updated createdir default size.

### LINKED ISSUE #329: createDir: Incorrect values are saved in reference_objects and break challenge
https://github.com/0chain/blobber/blob/7370c1e346733c3a31285bd0e75be1a87796c2ab/code/go/0chain.net/blobbercore/allocation/newfilechange.go#L89

It should be 

```
newDir.ActualFileSize = 0
newDir.Size = 0
newDir.NumBlocks = 0
````

Because it is new empty directory.

### LINKED ISSUE #441: duplicate dir creation is not giving any error
/zbox createdir --allocation 4157af127d56077124753022f00cf84fb09df87f66d6bc46f9a61a27d45ff851 --dirname test

if we execute createdir comand with same dir name then in database instead of giving an error it creates duplicate entries.

test     | /test                 | 4157af127d56077124753022f00cf84fb09df87f66d6bc46f9a61a27d45ff851
 test     | /test                 | 4157af127d56077124753022f00cf84fb09df87f66d6bc46f9a61a27d45ff851
 test     | /test                 | 4157af127d56077124753022f00cf84fb09df87f66d6bc46f9a61a27d45ff851
 test     | /test                 | 4157af127d56077124753022f00cf84fb09df87f66d6bc46f9a61a27d45ff851
 test     | /test                 | 4157af127d56077124753022f00cf84fb09df87f66d6bc46f9a61a27d45ff851
 test     | /test                 | 4157af127d56077124753022f00cf84fb09df87f66d6bc46f9a61a27d45ff851
 test     | /test                 | 4157af127d56077124753022f00cf84fb09df87f66d6bc46f9a61a27d45ff851
 test     | /test                 | 4157af127d56077124753022f00cf84fb09df87f66d6bc46f9a61a27d45ff851
 /        | /                     | 4157af127d56077124753022f00cf84fb09df87f66d6bc46f9a61a27d45ff851

### PR DESCRIPTION
**Pull request by princeparmar**
fixed size and block values in newly created dir.
added handling for duplicate dir creation when we try like `createdir --dirpath temp`

Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/blobber/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master BEFORE merging this PR

Associated PRs (Link as appropriate):
- zbox: N/A
- zwalelet: N/A
- gosdk: N/A
- 0chain: N/A
- system_test: https://github.com/0chain/system_test/pull/156

### CODE REVIEW

File: code/go/0chain.net/blobbercore/filestore/fs_store.go [hunk 1]
```diff
@@ -451,7 +451,7 @@ func (fs *FileFSStore) DeleteFile(allocationID string, contentHash string) error
 }
 
 func (fs *FileFSStore) CreateDir(dirName string) error {
```
Comment (cnlangzi) [inline code/go/0chain.net/blobbercore/filestore/fs_store.go]: CreateDir should be removed from `Store` interface

File: code/go/0chain.net/blobbercore/allocation/newfilechange.go [hunk 2]
```diff
@@ -94,13 +94,15 @@ func (nf *NewFileChange) CreateDir(ctx context.Context, allocationID, dirName, a
...
 	newDir.AllocationID = dirRef.AllocationID
 	newDir.MerkleRoot = nf.MerkleRoot
-	newDir.Name = dirName
-	newDir.Size = 2
-	newDir.NumBlocks = 2
+	newDir.Path = dirName
+	newDir.Size = 0
+	newDir.NumBlocks = 0
 	newDir.ParentPath = dirRef.Path
 	newDir.WriteMarker = allocationRoot
```
Comment (cnlangzi) [inline code/go/0chain.net/blobbercore/allocation/newfilechange.go:107]: I am not sure if allocationRoot is correct WriteMarker for it. what is allocationRoot here?
It seems is unnecessary on the `Mkdir`. WriteMarker seems is designed for transaction on blockchain.
please double check it.

**Review by cnlangzi — CHANGES_REQUESTED** [review]

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | fix/issue-155-remove-size-check-create-dir  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | bug/issue-329-incorrect-ref-value  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | fix/issue-155-remove-size-check-create-dir  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | bug/issue-329-incorrect-ref-value  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Review by cnlangzi — APPROVED** [review]

**Pull request merged**